apps/communications/contacts/js/match_service.js

```diff
@@ -1,39 +1,48 @@
 'use strict';
-/* global utils */
 
 (function(exports) {
   var matchService = {};
 
   const CONTACTS_APP_ORIGIN = location.origin;
-
-  var extensionFrame = document.createElement('iframe');
-  extensionFrame.id = 'match-extension';
-  document.body.appendChild(extensionFrame);
   
+  var extensionFrame = null;
```

Comment:
Should we still call it `extensionFrame`? Now we could use name that suits better the situation